test/sysstats/main.d

```diff
@@ -58,7 +58,7 @@ class MyApp : DaemonApp
         auto timer = new TimerEvent(
                 {
                     // wait until uptime advances, clock might be slower on VMs
-                    if (ProcVFS.getProcUptime() == uptime)
+                    if (ProcVFS.getProcUptime().uptime == uptime.uptime)
```

Comment:
What if this never changes, will the test never exit?

Reply:
I think it's fair not to optimize for the broken case, but to rely on the timeout on CI. Because, even the rounding is happening, you're still _using_ some CPU time in the timer handler/setting up the timer, so it should eventually happen.

Reply:
I would be more concerned about virtualized systems that always report uptime to be some fake value. But we can wait until such is actually encountered :)